src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/FeatureSwitchManager.cs

```diff
@@ -591,6 +591,12 @@ private bool TryGetConstantArgument(MethodIL methodIL, byte[] body, OpcodeFlags[
                         {
                             return true;
                         }
+                        else if (method.IsIntrinsic && method.Name is "get_IsValueType"
```

Comment:
`method.IsIntrinsic` can be de-duplicated with the existing case above.

Reply:
We need this to land in the `else` branch all the way below if it's an intrinsic but not the one we recognize.